device.go

```diff
@@ -86,6 +108,16 @@ func (d *device) InstallUpdate(image io.ReadCloser, size int64) error {
 		return err
 	}
 
+	// Make sure the file system is not mounted (MEN-2084)
+	if checkMounted(inactivePartition) != "" {
+		err = syscall.Unmount(inactivePartition, syscall.MNT_FORCE)
```

Comment:
We should not use `MNT_FORCE`, but rather a gentle unmount and fail if that doesn't work. Force-unmounting could mean that some process actually has open files there, in which case we should definitely fail instead.